src/locale/languages/zh-cn.js

```diff
@@ -0,0 +1,335 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const zhCn = {
```

Comment:
@andrewda Please forgive me. I have never seen this kind of abbreviation. Normal usages are `zhCN/zhTW/enUS` and so on. What do you think? Maybe we should revert it as `zhCN` and update `ptBr` as `ptBR`.

Reply:
Yea, I think `zhCN/ptBR` is more common as `CN/BR/US` are country abbreviations. `zhCn` or `enUs` looks weird to me. @andrewda ?